templates/contest/list.html

```diff
@@ -124,7 +124,7 @@
 {% macro contest_join(contest, request) %}
     {% if not request.in_contest %}
         <td>
-            {% if request.profile in contest.organizers.all() %}
+            {% if request.profile.id in contest.editor_ids or request.profile.id in contest.tester_ids %}
```

Comment:
I'm not sure if using `editor_ids` and `tester_ids` here is a good idea. Doesn't it cause a query regardless of `prefetch_related`? It might be a better to just use `contest.authors.all()`, `contest.curators.all()`, and `contest.testers.all()` directly.

Reply:
The ids should be marked with `@cached_property`, so maybe not(?), I'll have to test that

Reply:
Because we `prefetch_related`, `contest.authors.all()` is the better choice.